packages/db/src/schema.prisma

```diff
@@ -366,3 +372,27 @@ model Footnote {
 
   @@id([wordId, languageId])
 }
+
+/// Represents a gloss update for a particular language.
```

Comment:
Done.

Reply:
>  I don't think there's any problem, but I'll definitely avoid doing that in the future.

It will probably happen again, I've accidentally done it many times. All you have to do when that happens is click the "Unresolve conversation" button.